### ISSUE TITLE
Bot uses incorrect action space

### ISSUE THREAD

**Issue by SameerDalal** [SameerDalal/QuantumGo_Engine#6]
The bot is not able to play on the second board because the action space only considers one board. Since both boards can have different states, the action space should include the second board as well. 

Need to account for the fact that some moves in the action space may not be unique because playing an action on one board is copied to another.

**Comment by benjaminpjones** [comment]
Is there value in having the action space include both boards? It seems like it will increase the breadth of the game tree without improving the bot's understanding. If it has searched `board1:A1`, then it has essentially searched `board2:A1` assuming non-quantum stones.  For quantum stones, the same logic applies, just with different coordinates.

**Comment by SameerDalal** [comment]
I would agree with you. I think that as of now we don't need to increase the size of the game tree. If at some point the bot gets quite good at playing with just one of the boards, and there are no other major areas of improvement, then we could consider adding the second board into the action space as well.
